tensorboard/plugins/scalar/scalars_plugin_test.py

```diff
@@ -171,6 +171,21 @@ def test_scalars_with_scalars(self):
         self.assertEqual("application/json", response.headers["Content-Type"])
         self.assertEqual(self._STEPS, len(json.loads(response.get_data())))
 
+    def test_scalarsmulti_with_scalars(self):
+        server = self.load_server([self._RUN_WITH_SCALARS])
+        response = server.post(
+            "/data/plugin/scalars/scalarsmulti",
+            data={
+                "runs": json.dumps([self._RUN_WITH_SCALARS]),
```

Comment:
Nice test; thanks! Maybe include a run that doesn’t have scalar data
and/or a run that doesn’t have any data to test those edge cases, since
it’d be easy?

Reply:
Done.

Reply:
This works; thanks! I actually meant that it would suffice to just set
`runs` to `[_RUN_WITH_SCALARS, _RUN_WITH_HISTOGRAMS]` and keep the same
expected output. (That way, you’re also actually testing the multi-run
functionality of the `/scalars_multirun` route.) But having multiple
test cases is also perfectly fine with me if you prefer that.